include/libopencm3/stm32/fdcan.h

```diff
@@ -0,0 +1,1420 @@
+/** @defgroup fdcan_defines FDCAN defines
+@ingroup STM32H7_defines
```

Comment:
If you're putting this in a common directory, it can't be in the h7 file grouping please.

Reply:
That's fair. Cribbing from the `can.h` file, it looks like I should put this in the `STM32F_defines` group. ST claims that this IP is not specific to the H7...